css/main.css

```diff
@@ -1,3 +1,226 @@
 /* Add your CSS here */
 
-/* Dont' forget to link this file to your HTML in the <head> */
+.main {
+    width: 100%;
+    grid-area: main;
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: repeat(6, auto);
+    grid-row-gap: 20px;
+
+}
+
+.hero {
+    background-image: url(https://static.wixstatic.com/media/697bc8_ee5e15509a8c468f96ec8b15a13e0daf~mv2_d_2000_1809_s_2.jpg/v1/fill/w_980,h_792,al_b,q_85,usm_0.66_1.00_0.01/697bc8_ee5e15509a8c468f96ec8b15a13e0daf~mv2_d_2000_1809_s_2.webp);
+    width: 100%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    color: #fff;
+    text-align: center;
+    padding: 6em 0;
+    position: relative;
+    grid-column: 1/13;
+    grid-row: 1/2;
+}
+
+.hero_container {
+    display: flex;
+    flex-direction: column;
+
+}
+
+.hero_logo {
+    width: 40%;
+    height: auto;
+
+}
+
+.hero_logo_est {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+}
+
+.hero_logo_est h2 {
+    margin: auto;
+    font-size: 6vw;
+    border-bottom: 2px solid white;
+}
+
+.hero_text {
+    font-size: 8vw;
+}
+
+.hero_btn {
+    margin-top: 30px;
+
+    padding: 0.5rem 0.9rem;
+    color: white;
+    background-color: var(--red);
+    font-size: 5vw;
+    cursor: pointer;
+    display: inline-block;
+}
+
+
+/******************************************* Shopping */
+.shopping {
+    display: grid;
+    grid-column: 1/13;
+
+    grid-template-rows: auto auto auto;
+    grid-template-columns: 1fr;
+    grid-row-gap: 20px;
+}
+
+.shopping img {
+    width: 100%;
+    max-width: 100%;
+}
+
+
+.cacti_content,
+.plants_content,
+.succulents_content {
+    width: 100%;
+    text-align: center;
+    position: absolute;
+    top: 0;
+    left: 0;
+    padding: 5rem 0;
+
+}
+
+.shopping_btn {
+
+    background-color: var(--red);
+    color: white;
+    cursor: pointer;
+    border-radius: 5px;
+    padding: 0.5rem;
+
+}
+
+.shopping hr {
+    border: 4px solid var(--red);
+    margin: 10px auto;
+    background-color: var(--red);
+    width: 4%;
+
+}
+
+/* ******************************************Cacti */
+
+.cacti {
+    position: relative;
+    width: 100%;
+
+}
+
+.cacti_content h2 {
+    color: var(--red);
+    font-size: 2rem;
+
+}
+
+/**************************************************** Plants */
+
+.plants {
+    position: relative;
+    width: 100%;
+
+}
+
+.plants_content h2 {
+    color: var(--red);
+    font-size: 2rem;
+}
+
+/**************************************************** SUCCULANTS */
+
+.succulents {
+    position: relative;
+    width: 100%;
+}
+
+.succulents_content h2 {
+    color: var(--red);
+    font-size: 2rem;
+}
+
+/******************************************************* about_us */
+
+.about_us {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(7, 1fr);
```

Comment:
Your about_us images are being stretched as they are set to be 100% of their parent, there is something that you can change which will fix this, while the images still having height: 100%, can you find what that is?